File: chatcli/control/env.py
import os

class Env():
    def __init__(self):
        self.openai_api_key = os.getenv("OPENAI_API_KEY", default=None)
        self.openai_model = os.getenv("OPENAI_MODEL", default=None)
        self.config_path = os.getenv("CHATCLI_CONFIG", default=None)
        self.pre_injection = os.getenv("PRE_INJECTION", default=None)
        self.post_injection = os.getenv("POST_INJECTION", default=None)

File: control/env.go
package control

import (
	"os"
)

type Env struct {
	OpenAIKey     string
	OpenAIModel   string
	ConfigPath    string
	PreInjection  string
	PostInjection string
}

func NewEnv() *Env {
	return &Env{
		OpenAIKey:     os.Getenv("OPENAI_API_KEY"),
		OpenAIModel:   os.Getenv("OPENAI_MODEL"),
		ConfigPath:    os.Getenv("CHATCLI_CONFIG"),
		PreInjection:  os.Getenv("CHATCLI_PRE_INJECTION"),
		PostInjection: os.Getenv("CHATCLI_POST_INJECTION"),
	}
}
